nvflare/private/fed/simulator/simulator_server.py

```diff
@@ -134,3 +126,15 @@ def create_run_manager(self, workspace, job_id):
             client_manager=self.client_manager,
             handlers=self.runner_config.handlers,
         )
+
+    def wait_engine_run_complete(self, name, shutdown_grace_time=2.0, cleanup_grace_time=3.0):
```

Comment:
arg `shutdown_grace_time` is not used, we can remove it

Reply:
Don't need this wait_engine_run_complete() any more. Deleted.